export/light.py

```diff
@@ -96,7 +96,12 @@ def convert_luxcore_settings(exporter, obj, obj_key, depsgraph, luxcore_scene, t
         if light.luxcore.use_advanced:
             definitions["efficency"] = light.luxcore.efficacy
             definitions["power"] = light.luxcore.power
-            definitions["gain"] = [1, 1, 1]
+            # Set Gain to 0 if Power or Efficacy is 0
```

Comment:
These "Set Gain to 0 if Power or Efficacy is 0" comments are redundant, the if statement below contains exactly the same information.
I would either remove them or replace them with something that adds useful information, like an explanation that lines out the reason why this is done.